lib/ui/painting/image_decoder_no_gl_unittests.cc

```diff
@@ -0,0 +1,212 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/painting/image_decoder_no_gl_unittests.h"
+
+namespace flutter {
+namespace testing {
+
+// Tests are disabled for fuchsia.
```

Comment:
One of these tests just got moved over here, so it wasn't running for fuchsia anyway.